src/lib/Drawer/index.tsx

```diff
@@ -152,6 +152,7 @@ const Drawer: React.FunctionComponent<Props> = ({
 
   return (
     <Portal>
+      <>
```

Comment:
adding these seems to fix it - it wraps the children so it's one element and fixes teh issue. Modal and others didn't have this issue because they were naturally wrapped in divs.

Reply:
and just for a future note in case we come across it again this is the issue that it fixes: 
 This JSX tag's 'children' prop expects a single child of type 'ReactElement<any, string | ((props: any) => ReactElement<any, string | ... | (new (props: any) => Component<any, any, any>)> | null) | (new (props: any) => Component<any, any, any>)> | null', but multiple children were provided.
src/lib/Drawer/index.tsx

Reply:
that was my bad...